app/__init__.py

```diff
@@ -1,5 +1,9 @@
 from flask import Flask
 from config import config
+from flask.ext.bootstrap import Bootstrap
+
+
+bootstrap = Bootstrap()
```

Comment:
I'm still not convinced the explorer should be part of the API applications. We end up introducing a fair amount of front end dependency on something that isn't ever run in production. Having a separate application that we can easily point to any environment would keep things separate.